pkg/process/procutil/process_linux.go

```diff
@@ -89,8 +89,18 @@ func (p *Probe) Close() {
 	}
 }
 
+// StatsForPIDsWithPerm collects stats for processes that require extra permissions
+func (p *Probe) StatsForPIDsWithPerm(pids []int32, now time.Time) (map[int32]*Stats, error) {
```

Comment:
I don't know if I see a use-case where someone wants to flip between these two functions. Can we just configure whether this should be collected or not in the `Probe{}` itself?

Reply:
Updated. That's a good choice.